### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by brizaldi** [nnnggel/gCaptcha#3]
is this library support iOS?

**Comment by nnnggel** [comment]
Nope. This plugin depends on google SafetyNet which is in android ecosystem.
https://developer.android.com/training/safetynet/recaptcha

**Comment by brizaldi** [comment]
any recommendation what library to use for iOS?

**Comment by nnnggel** [comment]
https://cloud.google.com/recaptcha-enterprise/docs/instrument-ios-apps
